src/main/java/it/pagopa/ecommerce/payment/methods/controller/v1/PaymentMethodsController.java

```diff
@@ -241,6 +241,7 @@ public Mono<ResponseEntity<CalculateFeeResponseDto>> calculateFees(
     @Override
     public Mono<ResponseEntity<CreateSessionResponseDto>> createSession(
                                                                         String id,
+                                                                        String lang,
```

Comment:
This language will be used to populate NPG POST order/build language field into another PR?

Reply:
yes 
here 
https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/227